support-frontend/app/models/identity/responses/IdentityErrorResponse.scala

```diff
@@ -29,12 +29,14 @@ object IdentityErrorResponse {
   case class IdentityError(
       message: String,
       description: String,
+      context: String = "primaryEmailAddress",
```

Comment:
I'm not clear what this `IdentityError` object represents, but it looks generic enough to handle any Identity Error, so I'm not sure what impact adding a context here has?